contrib/google.golang.org/grpc/server.go

```diff
@@ -75,15 +76,32 @@ func StreamServerInterceptor(opts ...Option) grpc.StreamServerInterceptor {
 
 		// if we've enabled call tracing, create a span
 		if cfg.traceStreamCalls {
+			var methodKind string
+			if info != nil {
+				switch {
+				case info.IsServerStream && info.IsClientStream:
+					methodKind = methodKindBidiStream
+				case info.IsServerStream:
+					methodKind = methodKindServerStream
+				case info.IsClientStream:
+					methodKind = methodKindClientStream
+				}
+			}
+			var spanOpts []ddtrace.StartSpanOption
+			if methodKind != "" {
+				spanOpts = []ddtrace.StartSpanOption{ tracer.Tag(tagMethodKind, methodKind) }
+			}
 			var span ddtrace.Span
 			span, ctx = startSpanFromContext(
 				ctx,
 				info.FullMethod,
 				"grpc.server",
 				cfg.serviceName,
 				cfg.analyticsRate,
+				spanOpts...,
 			)
 			defer func() { finishWithError(span, err, cfg) }()
+
```

Comment:
Why the line?

Reply:
no reason at all. 😄

Reply:
Can we pls remove it?

Reply:
Yes, of course! Sorry, it's already done on my local copy.